library/kotlin/io/envoyproxy/envoymobile/EnvoyManifest.xml

```diff
@@ -2,7 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
           package="io.envoyproxy.envoymobile">
     <uses-sdk
-            android:minSdkVersion="9"
-            android:targetSdkVersion="27" />
+            android:minSdkVersion="21"
```

Comment:
is this the min version where this exists?

Reply:
The min version here is correct. We do want API 21 (Android L) as our min